packages/ProgressIndicators/src/ProgressBar/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+/** Props for ProgressBar */
+interface ProgressBarProps {
+  anime: boolean;
```

Comment:
Let's add `anime` to the documentation.
![giphy-downsized (1)](https://user-images.githubusercontent.com/4141527/65975763-f3654780-e423-11e9-8a07-3e2d8dda79c2.gif)

Reply:
cool